repository-ydb-v2/src/main/java/tech/ydb/yoj/repository/ydb/compatibility/YdbSchemaCompatibilityChecker.java

```diff
@@ -42,7 +43,7 @@
 public final class YdbSchemaCompatibilityChecker {
     private static final Logger log = LoggerFactory.getLogger(YdbSchemaCompatibilityChecker.class);
 
-    private final List<Class<? extends Entity>> entities;
+    private final List<TableDescriptor<? extends Entity>> descriptors;
```

Comment:
Here and in other places - generic parametr of TableDescriptor already <E extend<Entity<E>>>, because of it we can use just TableDescriptor<?>